https://%D0%BE%D0%BD%D0%BB%D0%B0%D0%B8%CC%86%D0%BD-%D1%87%D0%B8%D1%82%D0%B0%D1%82%D1%8C.%D1%80%D1%84 - https://онлайн-читать.рф – request error: Failed to parse: '%d0%be%d0%bd%d0%bb%d0%b0%d0%b8%cc%86%d0%bd-%d1%87%d0%b8%d1%82%d0%b0%d1%82%d1%8c.%d1%80%d1%84', label empty or too long
https://irr.ru - https://irr.ru – request error: HTTPSConnectionPool(host='irr.ru', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x103cc8d10>: Failed to resolve 'irr.ru' ([Errno 8] nodename nor servname provided, or not known)"))
https://beru.ru tiu.ru - https://beru.ru tiu.ru – request error: HTTPSConnectionPool(host='beru.ru%20tiu.ru', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10356b090>: Failed to resolve 'beru.ru%20tiu.ru' ([Errno 8] nodename nor servname provided, or not known)"))
https://riafan.ru - https://riafan.ru – request error: HTTPSConnectionPool(host='riafan.ru', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1032ed150>: Failed to resolve 'riafan.ru' ([Errno 8] nodename nor servname provided, or not known)"))
https://%D1%81%D1%82%D0%BE%D0%BF%D0%BA%D0%BE%D1%80%D0%BE%D0%BD%D0%B0%D0%B2%D0%B8%D1%80%D1%83%D1%81.%D1%80%D1%84 - https://стопкоронавирус.рф – request error: Failed to parse: '%d1%81%d1%82%d0%be%d0%bf%d0%ba%d0%be%d1%80%d0%be%d0%bd%d0%b0%d0%b2%d0%b8%d1%80%d1%83%d1%81.%d1%80%d1%84', label empty or too long
https://%D0%B0%D0%B2%D1%86.%D1%80%D1%84 - https://авц.рф – request error: HTTPSConnectionPool(host='%d0%b0%d0%b2%d1%86.%d1%80%d1%84', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x110c80f50>: Failed to resolve '%d0%b0%d0%b2%d1%86.%d1%80%d1%84' ([Errno 8] nodename nor servname provided, or not known)"))
https://rulit.online - https://rulit.online – request error: HTTPSConnectionPool(host='rulit.online', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1045f4690>: Failed to resolve 'rulit.online' ([Errno 8] nodename nor servname provided, or not known)"))
https://%D0%BC%D1%8B%D0%B2%D0%BC%D0%B5%D1%81%D1%82%D0%B52020.%D1%80%D1%84 - https://мывместе2020.рф – request error: HTTPSConnectionPool(host='%d0%bc%d1%8b%d0%b2%d0%bc%d0%b5%d1%81%d1%82%d0%b52020.%d1%80%d1%84', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x104982cd0>: Failed to resolve '%d0%bc%d1%8b%d0%b2%d0%bc%d0%b5%d1%81%d1%82%d0%b52020.%d1%80%d1%84' ([Errno 8] nodename nor servname provided, or not known)"))
https://futurerussia.gov.ru - https://futurerussia.gov.ru – request error: HTTPSConnectionPool(host='futurerussia.gov.ru', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10386b910>: Failed to resolve 'futurerussia.gov.ru' ([Errno 8] nodename nor servname provided, or not known)"))
https://%D0%BC%D0%BE%D0%B8%D0%B4%D0%BE%D0%BA%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D1%8B.%D1%80%D1%84 - https://моидокументы.рф – request error: Failed to parse: '%d0%bc%d0%be%d0%b8%d0%b4%d0%be%d0%ba%d1%83%d0%bc%d0%b5%d0%bd%d1%82%d1%8b.%d1%80%d1%84', label empty or too long
https://%D0%BF%D1%80%D0%B5%D0%B7%D0%B8%D0%B4%D0%B5%D0%BD%D1%82%D1%81%D0%BA%D0%B8%D0%B5%D0%B3%D1%80%D0%B0%D0%BD%D1%82%D1%8B.%D1%80%D1%84 - https://президентскиегранты.рф – request error: Failed to parse: '%d0%bf%d1%80%d0%b5%d0%b7%d0%b8%d0%b4%d0%b5%d0%bd%d1%82%d1%81%d0%ba%d0%b8%d0%b5%d0%b3%d1%80%d0%b0%d0%bd%d1%82%d1%8b.%d1%80%d1%84', label empty or too long
https://%D0%BE%D0%BD%D0%BB%D0%B0%D0%B8%CC%86%D0%BD%D0%B8%D0%BD%D1%81%D0%BF%D0%B5%D0%BA%D1%86%D0%B8%D1%8F.%D1%80%D1%84 - https://онлайнинспекция.рф – request error: Failed to parse: '%d0%be%d0%bd%d0%bb%d0%b0%d0%b8%cc%86%d0%bd%d0%b8%d0%bd%d1%81%d0%bf%d0%b5%d0%ba%d1%86%d0%b8%d1%8f.%d1%80%d1%84', label empty or too long
https://%D1%83%D1%80%D0%BE%D0%BA.%D1%80%D1%84 - https://урок.рф – request error: HTTPSConnectionPool(host='%d1%83%d1%80%d0%be%d0%ba.%d1%80%d1%84', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1032bd9d0>: Failed to resolve '%d1%83%d1%80%d0%be%d0%ba.%d1%80%d1%84' ([Errno 8] nodename nor servname provided, or not known)"))
https://%D0%B4%D0%BE%D1%81%D1%82%D1%83%D0%BF%D0%B2%D1%81%D0%B5%D0%BC.%D1%80%D1%84 - https://доступвсем.рф – request error: HTTPSConnectionPool(host='%d0%b4%d0%be%d1%81%d1%82%d1%83%d0%bf%d0%b2%d1%81%d0%b5%d0%bc.%d1%80%d1%84', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106b80e10>: Failed to resolve '%d0%b4%d0%be%d1%81%d1%82%d1%83%d0%bf%d0%b2%d1%81%d0%b5%d0%bc.%d1%80%d1%84' ([Errno 8] nodename nor servname provided, or not known)"))
https://money.yandex.ru - https://money.yandex.ru – request error: HTTPSConnectionPool(host='money.yandex.ru', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107923b10>: Failed to resolve 'money.yandex.ru' ([Errno 8] nodename nor servname provided, or not known)"))
https://kodeks.rusudact.ru - https://kodeks.rusudact.ru – request error: HTTPSConnectionPool(host='kodeks.rusudact.ru', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x102f91450>: Failed to resolve 'kodeks.rusudact.ru' ([Errno 8] nodename nor servname provided, or not known)"))
